addons/l10n_pe/demo/l10n_pe_demo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="partner_demo_company_pe" model="res.partner">
+        <field name="name">PE Company</field>
+        <field name="vat">20557912879</field>
+        <field name="street">Calle Las Tortugas</field>
+        <field name="city">Chorrillos</field>
+        <field name="country_id" ref="base.pe"/>
+        <field name="state_id" ref="base.state_pe_25"/>
+        <field name="zip">15067</field>
+        <field name="phone">+51 912 345 678</field>
+        <field name="email">info@company.peexample.com</field>
+        <field name="website">www.peexample.com</field>
+    </record>
+    <record id="demo_company_pe" model="res.company">
+        <field name="name">PE Company</field>
+        <field name="partner_id" ref="partner_demo_company_pe"/>
+    </record>
+
+    <record id="res_partner_1" model="res.partner">
```

Comment:
I suppose l10n_pe.res_partner_1 will be different than the base one then.  But another xml_id might be more logical.